packages/react-widgets/src/widgets/TimeSeriesWidget.js

```diff
@@ -81,6 +81,7 @@ const EMPTY_ARRAY = [];
  * @param  {string[]=} [props.palette] - Optional palette
  * @param  {object} [props.wrapperProps] - Extra props to pass to [WrapperWidgetUI](https://storybook-react.carto.com/?path=/docs/widgets-wrapperwidgetui--default).
  * @param  {object} [props.noDataAlertProps] - Extra props to pass to [NoDataAlert]().
+ * @param  {object} [props.intlConfig] - Object with intl configuration. If not provided, default messages will be used.
```

Comment:
please use `{object=}` to indicate that this is [optional parameter](https://stackoverflow.com/a/29441515)

(not necessarily here, as there is proper .d.ts, but mainly for other widgets and places when you add optional param